blueprint/core/blueprint_ViewManager.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+  ==============================================================================
+
+    blueprint_ViewManager.cpp
+    Created: 21 May 2020 15:32:00am
+
+  ==============================================================================
+*/
+
+#include "blueprint_ViewManager.h"
+
+namespace blueprint
+{
+    //TODO: Have the root ShadowView owned by ViewManager or have ReactApplicationRoot own this
```

Comment:
@nick-thompson  This is probably the key discussion point.

Reply:
I like how you have it now– ViewManager seems like the right spot to own all shadow views